src/main/java/frc/robot/Robot.java

```diff
@@ -460,15 +460,18 @@ private void executeIntakeStateMachine() {
         
         mIntake.resetOvercurrentCooldown();
         break;
+      case BARF:
```

Comment:
How do we ever get out of this state? Seems like we should transition to IDLE after a pre-configured amount of time.

Reply:
Done